go/src/koding/klient/terminal/command.go

```diff
@@ -31,6 +40,22 @@ func init() {
 	}
 }
 
+func guessTerm() string {
+	terms := [][2]string{
+		{"xterm-256color", "/usr/share/terminfo/x/xterm-256color"},
+		{"xterm-256color", "/usr/share/terminfo/78/xterm-256color"},
```

Comment:
In fact 78 is hex code for `x` - under Linux terminfo files are grouped in directories after the first letter, under macOS - after hex code value of the first letter.